Task2/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Task2;
+
+public class Program
+{
+    public static (string, int) Bwt(string str)
```

Comment:
```suggestion
    public static (string result, int endPosition) Bwt(string str)
```
Для упрощения читаемости кода элементам кортежей нужно давать имена, раз уж язык это позволяет.